packages/commonwealth/client/scripts/helpers/ContractHelpers/NamespaceFactory.ts

```diff
@@ -152,6 +152,44 @@ class NamespaceFactory extends ContractBase {
     }
     return txReceipt;
   }
+
+  async newContest(
+    namespaceName: string,
+    contestInterval: number,
```

Comment:
do we need interval here if this method is specific for non recurring contests? Should we default to 0 if the contest is non recurring?

Reply:
yes I will implement the above mentioned missed commit like this, good idea